rclcpp_generic/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rclcpp_generic</name>
+  <version>0.1.0</version>
+  <description>Communication between nodes with the message type chosen at runtime</description>
+  <maintainer email="nikolai.morin@apex.ai">Nikolai Morin</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>ament_index_cpp</depend>
```

Comment:
Some of these can probably be `<build_depend>` only, right?

Reply:
Some folks do the doubled `build_depend` and `exec_depend` for build dependencies that might be shared libraries - though this is an ok way to start as it is definitely correct, though it is looser - you can tighten up the definitions later when there is a motivation for it. That's just my take, though - someone else might have a diffferent opinion